app/src/main/res/values-de/arrays.xml

```diff
@@ -8,20 +8,8 @@
         <item>Semi-transparent Dunkel</item>
         <item>Transparent Dunkel</item>
     </string-array>
-    <string-array name="themesValues">
-        <item>light</item>
-        <item>semi-transparent</item>
-        <item>transparent</item>
-        <item>dark</item>
-        <item>semi-transparent-dark</item>
-        <item>transparent-dark</item>
-    </string-array>
     <string-array name="sortAppListEntries">
         <item>Von A nach Z</item>
         <item>Von Z nach A</item>
     </string-array>
-    <string-array name="sortAppListValues">
-        <item>alphabetical</item>
-        <item>invertedAlphabetical</item>
-    </string-array>
```

Comment:
Well, they somehow are... I discovered that I can use the translatable=false flahg to make this more explicit, I'll probably do this on master